R/uvhydrograph-render.R

```diff
@@ -327,7 +335,13 @@ createSecondaryPlot <- function(uvInfo, secondarySeriesList,
   
   invertPlot <- secondarySeriesList[['inverted']]
   
-  lims <- calculateLims(secondarySeriesList[['corrected']][['points']])
+  if(!isEmptyOrBlank(secondarySeriesList[['corrected']][['points']][['value']])){
+    lims <- calculateLims(secondarySeriesList[['corrected']][['points']])
+    corrEstPoints <- secondarySeriesList[['corrected']][['points']][['value']]
+  } else if(secondarySeriesList[['useEstimated']]){
+    lims <- calculateLims(secondarySeriesList[['estimated']][['points']])
+    corrEstPoints <- secondarySeriesList[['estimated']][['points']][['value']]
+  }
```

Comment:
make a note of this in the new lims ticket, https://github.com/USGS-R/repgen/pull/726/files#r126185926